packages/ember-glimmer/lib/syntax/outlet.js

```diff
@@ -116,6 +116,7 @@ class AbstractOutletComponentManager {
   }
 
   didCreateElement() {}
+  didRenderLayout() {}
```

Comment:
Note: I am dropping the bounds on `{{outlet}}` and `{{render}}` currently because there are no components to attach them to, but we'll likely have to put it somewhere (e.g. the controller) for the inspector to work